.config/typos.toml

```
@@ -14,6 +14,9 @@ exat = "exat"
 optin = "optin"
 smove = "smove"
 
+[default.ignore-words]
+"Parth"
```

I don't know the Toml syntax but this appears to follow the pattern used for other rules.

```suggestion
parth = "Parth"
```

---

Yeah, I also don't see ignore-words as one of the options supported on https://github.com/crate-ci/typos/blob/master/docs/reference.md

---

Well now it passed...

---

let me try to push the typo and test it

---

No. It didn't work :(